services/reddit/user-karma.service.js

```diff
@@ -0,0 +1,96 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { nonNegativeInteger } = require('../validators')
+const { metric } = require('../text-formatters')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  data: Joi.object({
+    link_karma: nonNegativeInteger,
+    comment_karma: nonNegativeInteger,
+  }).required(),
+}).required()
+
+module.exports = class RedditUserKarma extends BaseJsonService {
+  static get category() {
+    return 'social'
+  }
+
+  static get route() {
+    return {
+      base: 'reddit/user-karma',
+      pattern: ':variant(link|comment|combined)/:user',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Reddit User Karma',
+        namedParams: { variant: 'combined', user: 'example' },
+        staticPreview: {
+          label: 'combined karma',
+          message: 56,
+          color: 'red',
+          style: 'social',
+        },
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'reddit karma',
+      namedLogo: 'reddit',
+    }
+  }
+
+  static render({ variant, karma, user }) {
+    const label =
+      variant === 'combined'
+        ? `u/${user} karma`
+        : `u/${user} karma (${variant})`
+    return {
+      label: `${label}`,
```

Comment:
Can just use `label` directly here
```suggestion
      label,
```